metadata-ingestion/build.gradle

```diff
@@ -12,7 +12,8 @@ if (!project.hasProperty("extra_pip_requirements")) {
 }
 
 task checkPythonVersion(type: Exec) {
-  commandLine python_executable, '-c', 'import sys; assert sys.version_info >= (3, 7)'
+  commandLine python_executable, '-c',
+    'import sys; assert (3, 11) > sys.version_info >= (3, 7), f"Python version {sys.version_info[:2]} not allowed"'
```

Comment:
Change message to "Is too old"

Reply:
What do you mean? If someone is using 3.11 it won't be too old but too new.

Reply:
oh i missed the 3.11 part - then this is fine